docs/queues.txt

```diff
@@ -9,24 +9,28 @@ Queues
    :values: tutorial
 
 .. meta::
-   :keywords: php framework, odm, code example
+   :keywords: php framework, odm, code example, jobs
 
-If you want to use MongoDB as your database backend for Laravel Queue, change
-the driver in ``config/queue.php``:
+To use MongoDB as your database for Laravel Queue, change
+the driver in your application's ``config/queue.php`` file:
 
 .. code-block:: php
 
    'connections' => [
        'database' => [
            'driver' => 'mongodb',
-           // You can also specify your jobs specific database created on config/database.php
+           // You can also specify your jobs-specific database
+           // in the config/database.php file
            'connection' => 'mongodb',
-           'collection' => 'jobs',
-           'queue' => 'default',
-           'retry_after' => 60,
+           // 'collection' => 'jobs',
+           // 'queue' => 'default',
+           // 'retry_after' => 60,
```

Comment:
S: It's a little unclear why these fields are commented out - are they optional? I think either leaving them uncommented and highlighting line 24 or deleting them altogether would be better

Reply:
Based on the table below, `collection` and `queue` are required so maybe leaving uncommented and highlighting line 24 is the way to go